circle.yml

```diff
@@ -5,9 +5,9 @@ test:
   pre:
     - ? |
           # make the build-server and i18n string data in parallel
-          if [[ "$CIRCLE_NODE_INDEX" == 0 ]]; then NODE_ENV=test make build-server; fi
+          if [[ "$CIRCLE_NODE_INDEX" == 0 ]]; then NODE_ENV=test npm run build-server; fi
```

Comment:
Can we revert changes made to Circle CI config file and put them in the follow up PR?